person.rb

```diff
@@ -1,18 +1,31 @@
-class Person
+require_relative 'nameable'
```

Comment:
- [ ] Hi, you did an awesome job by implementing the changes after your first code reviewer's review, but it seems like you still have the issue related to the `library` name. Your library name is `namable` not **nameable**, so either changer your file `namable` to `nameable.rb` or fix the `typo` in the `person.rb` file. You can see I got the same error

![image](https://github.com/waelelsafty07/oop-school/assets/66476329/b82b89e0-ea37-40f1-962d-40f89ed63fe7)